force-app/main/default/classes/SFDCStopTeamController.cls

```diff
@@ -125,6 +125,14 @@ public class SFDCStopTeamController {
                 'https://trailblazer.me/id/dnaskar'
             )
         );
+        sfdcstopContributors.add(
+            new Contributor(
+                'Mohammad Rafi',
+                'https://pbs.twimg.com/profile_images/1208356452160589824/qLtb5g9P_400x400.jpg',
```

Comment:
Hi @M0195 can you add your image here instead of mine? Rest everything is perfect in the PR. I'll merge it.

Reply:
I have added

Reply:
Perfect Thanks!